Terminal.Gui/Drawing/W3CColors.cs

```diff
@@ -0,0 +1,171 @@
+﻿namespace Terminal.Gui;
+
+/// <summary>
+/// Helper class that resolves w3c color names to their hex values
+/// Based on https://www.w3schools.com/colors/color_tryit.asp
+/// </summary>
+public class W3CColors : IColorNameResolver
+{
+    Dictionary<string,Color> _knownColors=new Dictionary<string,Color>(StringComparer.InvariantCultureIgnoreCase)
```

Comment:
Shouldn't these be in a string resource. Not for loc reasons, but to minimize memory use when this class is instantiated? 

I think the `new Color("#xxxx")` ends up as code, which is less concerning, but it would be more efficient if these were specified as ints and not strings?